### PR TITLE
MARS-274: Detect device type

### PR DESCRIPTION
**Pull request by edwinf-barreram**
Detect device type to set the loan card query limit.

https://user-images.githubusercontent.com/102815442/197653310-9d4dd5a1-e44e-4dc5-ae8e-28a74d2c2c85.mov

### CODE REVIEW

**Review by emuvente — APPROVED** [review]
👍 Looks good!

File: src/components/Contentful/HomePage/NewHomeLoansByCategoryGrid.vue [hunk 1]
```diff
@@ -130,11 +133,19 @@ export default {
...
+
+		let loanValues;
+		if (isDesktop) {
+			loanValues = loanChannel?.loans?.values;
+		} else {
+			loanValues = Array(loanLimit).fill({ id: 0 });
+			loanValues[0] = loanChannel?.loans?.values[0];
+		}
+
+		// loanValues[0] = isDesktop ? loanChannel?.loans?.values : loanChannel?.loans?.values[0];
```
Comment (dyersituations) [inline src/components/Contentful/HomePage/NewHomeLoansByCategoryGrid.vue:148]: Remove commented out code?

**Review by mcstover — APPROVED** [review]

File: src/components/Contentful/HomePage/NewHomeLoansByCategoryGrid.vue [hunk 2]
```diff
@@ -55,7 +55,7 @@ const loanCategoryPrefetchQuery = gql`
 
 export default {
 	name: 'NewHomeLoansByCategoryGrid',
-	inject: ['apollo', 'cookieStore'],
+	inject: ['apollo', 'cookieStore', 'device'],
```
Comment (dyersituations) [inline src/components/Contentful/HomePage/NewHomeLoansByCategoryGrid.vue:58]: Just out of curiosity, where is this `device` being defined?

**Review by dyersituations — APPROVED** [review]

File: src/components/Contentful/HomePage/NewHomeLoansByCategoryGrid.vue (hunk 2 above)
Comment (mcstover) [reply src/components/Contentful/HomePage/NewHomeLoansByCategoryGrid.vue:58]: check vue-middleware

**Pull request merged**